src/content-helper/dashboard-widget/components/top-posts-filters.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { FilterSelectMetric, FilterSelectTime } from '../../common/filter-select';
+
+/**
+ * Defines the props structure for TopPostsFilters.
+ *
+ * @since 3.10.0
+ */
+interface TopPostsFiltersProps {
+	selectedPeriod: string;
+	onPeriodChange: ( event: React.ChangeEvent<HTMLSelectElement> ) => void;
+	selectedMetric: string;
+	onMetricChange: ( event: React.ChangeEvent<HTMLSelectElement> ) => void;
+}
+
+/**
+ * Returns a div element containing filter controls for the Top Posts widget.
+ *
+ * @since 3.10.0
+ *
+ * @param {TopPostsFiltersProps} props The component's props.
+ */
+export function TopPostsFilters(
```

Comment:
It will be more simpler and maintanable if we don't make this as a separate component. It does makes sense if we have some logic inside the component but these is just a simple wrapper.

Reply:
Will remove.